docs/articles/documentation/guides/basic-guides/run-tests.md

```diff
@@ -0,0 +1,633 @@
+
+## Deploy the Tested App
+
+TestCafe can execute a specified shell command before it starts tests. For instance, you can run a command that starts a local web server and deploys the tested app. TestCafe automatically terminates the process when tests are finished.
+
+Use the [-a (--app)](../../reference/command-line-interface.md#-a-command---app-command) CLI flag or the [runner.startApp](../../reference/api/runner/startapp.md) API method to provide a command:
+
+```sh
+testcafe chrome ./my-tests/ --app "node server.js"
+```
+
+```js
+await runner
+    .browsers('chrome')
+    .src('./my-tests/')
+    .startApp('node server.js')
+    .run();
+```
+
+*Related configuration file property:* [appCommand](../../reference/configuration-file.md#appcommand)
+
+TestCafe delays tests to allow the shell command to execute. The default timeout is *1000* milliseconds. Use the [--app-init-delay](../../reference/command-line-interface.md#--app-init-delay-ms) CLI flag or a [runner.startApp](../../reference/api/runner/startapp.md) parameter to specify the timeout value.
+
+```sh
+testcafe chrome ./my-tests/ --app "node server.js" --app-init-delay 4000
+```
+
+```js
+await runner
+    .browsers('chrome')
+    .src('./my-tests/')
+    .startApp('node server.js', 4000)
+    .run();
+```
+
+*Related configuration file property:* [appInitDelay](../../reference/configuration-file.md#appinitdelay)
+
+## Provide a Proxy URL
+
+If your network uses a proxy to access the internet, specify the proxy URL to TestCafe. Use the [--proxy](../../reference/command-line-interface.md#--proxy-host) command line argument or the [runner.useProxy](../../reference/api/runner/useproxy.md) API method:
+
+```sh
+testcafe chrome ./my-tests/ --proxy proxy.mycompany.com
+```
+
+```js
+await runner
+    .browsers('chrome')
+    .src('./my-tests/')
+    .useProxy('proxy.mycompany.com')
+    .run();
+```
+
+*Related configuration file property:* [proxy](../../reference/configuration-file.md#proxy)
+
+You can also specify URLs that should be accessed directly. Pass the list of URLs in the [--proxy-bypass](../../reference/command-line-interface.md#--proxy-bypass-rules) command line argument or a [runner.useProxy](../../reference/api/runner/useproxy.md) parameter:
+
+```sh
+testcafe chrome ./my-tests/ --proxy proxy.corp.mycompany.com --proxy-bypass localhost:8080
+```
+
+```js
+await runner
+    .browsers('chrome')
+    .src('./my-tests/')
+    .useProxy('proxy.corp.mycompany.com', 'localhost:8080')
+    .run();
+```
+
+*Related configuration file property:* [proxyBypass](../../reference/configuration-file.md#proxybypass)
+
+## Live Mode
+
+Live mode ensures the TestCafe process and browsers remain active while you work on tests. You can see test results instantly because the tests are restarted when you make changes.
+
+![Live mode demonstrated in action](../../../images/testcafe-live.gif)
+
+> This feature replaces the `testcafe-live` module. This module is now deprecated.
```

Comment:
> This feature replaces the `testcafe-live` module. This module is now deprecated.

This feature replaces the deprecated `testcafe-live` module.

?

Reply:
Yes, I was just trying to be official as hell.